blazor/ClientSideExample/ClientSideExample/Views/FetchDataView.razor.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Reactive.Linq;
+using System.Threading.Tasks;
+using ClientSideExample.ViewModels;
+using Microsoft.AspNetCore.Components;
+using ReactiveUI;
+
+
+namespace ClientSideExample.Views
+{
+    public partial class FetchDataView : IViewFor<FetchDataViewModel>
+    {
+        [Inject]
+        public FetchDataViewModel ViewModel { get; set; }
+
+
+        object IViewFor.ViewModel
+        {
+            get => ViewModel;
+            set => ViewModel = (FetchDataViewModel)value;
+        }
+
+        protected override async Task OnInitializedAsync()
+        {
+           await ViewModel.LoadForecasts.Execute().SubscribeOn(RxApp.MainThreadScheduler);
```

Comment:
Ditto

Reply:
I'll test it it without but wasm doesn't like threading.

Reply:
`'IDisposable' does not contain a definition for 'GetAwaiter' and no accessible extension method 'GetAwaiter' accepting a first argument of type 'IDisposable' could be found (are you missing a using directive or an assembly reference?)`